src/desktop/DesktopUtils.js

```diff
@@ -162,6 +161,8 @@ function checkForAdminStatus(): Promise<boolean> {
 }
 
 function getLockFilePath() {
+	// don't get temp dir path from DesktopDownloadManager because we want the lockfile to persist while the app is open,
```

Comment:
This is unclear to me:
 - Is temp dir for temp files the same dir as the one with lock file? if yes, it shouldn't be
 - Why getting it from DownloadManager should delete the directory? doesn't it create it instead?

Reply:
1: no it's not, I accidentally made it be when I first did the changes, and realised what I'd done so I wrote a comment to explain the issue
2: getting the directory from the download manager doesn't destroy the directory, but the directory path that the download manager returns will be destroyed at some point during the app lifecycle, so the lockfile goes in `/tmp` and the download manager always gives `/tmp/tutanota/<whatever trailing path>` (basically as it already was)